client/public-ie11/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@luigi-project/client-ie11",
-  "version": "1.5.0",
+  "version": "1.5.1",
```

Comment:
is there a reason why we are switching to **1.5.1** here. Probably sth I am not aware of? If not, then I would change it back, or even better merge the master branch on top of this one so it contains the 1.6.0 version instead in all package.json's

Reply:
I guess it had been changed when i was ...  by the way, 1.6.0 is out... I will replace it with 1.6.0